src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/generalapplicationdirections/GeneralApplicationDirectionsNewEventAboutToSubmitHandler.java

```diff
@@ -184,14 +189,45 @@ private GeneralApplicationCollectionData setStatusForNonCollAndBulkPrintDocument
         //Generate GAD document
         CaseDocument caseDocument = service.generateGeneralApplicationDirectionsDocument(userAuthorisation, finremCaseDetails);
 
-        CaseDetails caseDetails = finremCaseDetailsMapper.mapToCaseDetails(finremCaseDetails);
+        setGeneralApplicationInformation(items, finremCaseDetails, caseDocument, status, bulkPrintDocuments, userAuthorisation);
 
-        items.setGeneralApplicationDirectionsDocument(caseDocument);
-        String gaElementStatus = status != null ? status : items.getGeneralApplicationStatus();
+        if (items.getGeneralApplicationSender().getValue().getCode().equalsIgnoreCase(INTERVENER1)
+            || items.getGeneralApplicationSender().getValue().getCode().equalsIgnoreCase(INTERVENER2)
+            || items.getGeneralApplicationSender().getValue().getCode().equalsIgnoreCase(INTERVENER3)
+            || items.getGeneralApplicationSender().getValue().getCode().equalsIgnoreCase(INTERVENER4)) {
+            gaService.updateIntervenerDirectionsOrders(items, finremCaseDetails);
```

Comment:
I cannot find any AC about interveners logic in your ticket.  Could you update the AC if it's required please?